baselines/logging/src/v2/logging_service_v2_client.ts.baseline

```diff
@@ -301,12 +306,14 @@ export class LoggingServiceV2Client {
           throw err;
         });
 
+      const descriptor =
+        this.descriptors.page[methodName] ||
+        this.descriptors.batching[methodName] ||
```

Comment:
FYI: this fixes the second part of the bug (note that the batching descriptor was never passed down to `createApiCall` before).